mars/ligation.lfr

```diff
@@ -0,0 +1,26 @@
+module ligation (
+    finput B, C, D, E, F,
+    foutput g_out,
+    control c1, c2
+);
+
+    // mix T4 DNA lygase buffer, vector DNA, insert DNA, Nuclease-free water, and T4 DNA lygase
+    flow mixture;
+    assign mixture = 500%B + 500%C + 500%D + 500%E + 500%F;
+
+    // incubate
+    flow incubated;
+    #MAP 'Incubator' '~'
```

Comment:
This needs to be inside the distribute block right above the assign (I think), I don't know if this will work atm, but its good you found this

Reply:
it doesnt work/
code:
```
    flow incubated;

    distribute@(c1) begin 
        #MAP "INCUBATOR" "~"
        if (c1 == 1)
            incubated <= ~mixture;
    end
```

error:
```
line 15:8 mismatched input '#MAP' expecting {'if', 'case', '{', ID}
line 16:8 mismatched input 'if' expecting 'endmodule'
Stopping compiler because of syntax errors
```

Reply:
You should create an issue for this